packages/jellyfish-api-core/__tests__/category/blockchain/getMempoolAncestors.test.ts

```diff
@@ -0,0 +1,93 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { Testing } from '@defichain/jellyfish-testing'
+import BigNumber from 'bignumber.js'
+
+describe('getMempoolAncestors', () => {
```

Comment:
For tests (`it('')` and `describe('')`), they should not be inter-dependent but self-sufficient.

Given that we use the same Container for many of the tests in the same file, it becomes challenging to write clean tests that are non-inter-dependent. 

And why do we use the same container — well simplify for test velocity and speed. Since spinning up multiple containers compared to one is much slower. 

You should break the `describe` and `it` into independent units that can be run on its own without relying on a sequential order. The `describe` block must describe a container with the condition that is scoped to a scenario. The scenario must not be created by a `it` condition (those are called side-effects).

> I think there are plentiful literature of how tests are written in this repo, but it's very difficult to find them for your reference since they're buried in PR comments.